src/ProfilesPage/ProfilePage.js

```diff
@@ -86,6 +92,30 @@ class ProfilePage extends Component {
     this.setState({editUserEnable: editable});
   }
 
+  onSuspend() {
+    const id = this.props.userInfo.id
+    var suspended = this.props.userInfo.suspended
+    const data = { id, suspended }
+    this.props.suspendUser(data)
+    this.setState({suspendedState: true})
+    console.log(sessionStorage.getItem('userId'))
+  }
+
+  onUnsuspend() {
+    const id = this.props.userInfo.id
```

Comment:
use destructuring i.e. ` const { userInfo } = this.props;` in line above

Reply:
done